Json.More/JsonElementExtensions.cs

```diff
@@ -205,8 +206,8 @@ public static JsonElement AsJsonElement(this decimal value)
 		/// <remarks>This is a workaround for lack of native support in the System.Text.Json namespace.</remarks>
 		public static JsonElement AsJsonElement(this double value)
 		{
-			var doc = JsonDocument.Parse($"{value}");
-			return doc.RootElement;
+			using var doc = JsonDocument.Parse(value.ToString(CultureInfo.InvariantCulture));
```

Comment:
`value.ToString(CultureInfo.InvariantCulture)` prevents unexpected encoding in different locales, e.g. `,` instead of `.`.
https://replit.com/@ddunkin/NET-doubleToString